DEVELOPERS.md

```diff
@@ -0,0 +1,20 @@
+# Information for Developers
+
+## `Images` Folder
+
+In the `Images` folder, you'll find PDFs of all the graphical assets as well as the `.afdesign` files from the program I created them with, Affinity Designer. We use [Font Awesome](https://fontawesome.com) to ensure a consistent design language across the website and app.
```

Comment:
You switch between `I` and `we`. I would use indirect speech to avoid this; like this
`The images folder contains all graphical assets; created using the Affinity Designer (.afdesign files) and then exported to pdf.`

Reply:
Good catch! I tried the passive voice, but couldn't get it to sound right, so I ended up just going with "we" consistently.